openapi.yml

```diff
@@ -55,16 +93,80 @@ components:
       properties:
         id:
           type: string
+        type:
```

Comment:
`type` may be valid as a property name, but it is a little confusing given that it is used elsewhere in the spec to mean something else. Can we come up with an alternative property name?